app/templates/plant_variety/edit_form.html

```diff
@@ -102,6 +89,24 @@
                     Care type</label>
                 {{ form.care_type(class_="w-full rounded-lg") }}
             </div>
+            <div class="flex items-start mb-6">
+                <div class="flex items-center h-5">
+                    <input name="is_moisture_loving"
+                        type="checkbox" "{% if form.is_moisture_loving.data %} checked {% endif %}"
+                        class="w-4 h-4 border border-gray-300 rounded bg-gray-50 focus:ring-3 focus:ring-blue-300 dark:bg-gray-700 dark:border-gray-600 dark:focus:ring-blue-600 dark:ring-offset-gray-800">
+                </div>
+                <label for="is_moisture_loving" class="ms-2 text-sm font-medium text-gray-900 dark:text-gray-300">Is
+                    moisture loving</label>
+            </div>
+            <div class="flex items-start mb-6">
+                <div class="flex items-center h-5">
+                    <input name="can_plant_indoors"
+                        type="checkbox" "{% if form.can_plant_indoors.data %} checked {% endif %}"
+                        class="w-4 h-4 border border-gray-300 rounded bg-gray-50 focus:ring-3 focus:ring-blue-300 dark:bg-gray-700 dark:border-gray-600 dark:focus:ring-blue-600 dark:ring-offset-gray-800">
+                </div>
+                <label for="can_plant_indoors" class="ms-2 text-sm font-medium text-gray-900 dark:text-gray-300">Can
+                    plant indoors</label>
+            </div>
             <div class="flex items-start mb-6">
```

Comment:
Add required attribute
for adding "*" symbol in input label use:
label:has(+ *:required):after,
label:has(~ div *:required):after {
  content: ' *';
  color: red;
}